src/components/expression/expression-renderer.js

```diff
@@ -87,6 +88,12 @@ genotet.ExpressionRenderer = function(container, data) {
   this.LABEL_MARGIN_ = 10;
 
   /**
+   * The difference between the labels and the heatmap.
+   * @private {number}
+   */
+  this.LABEL_DIFFERENCE_ = 10;
```

Comment:
These constants are used throughout the same class, and may be defined under prototype.